src/py21cmmc/likelihood.py

```diff
@@ -185,7 +185,11 @@ def _read_data(self):
                     )
                 )
             else:
-                data.append(dict(np.load(fl, allow_pickle=True)))
+                try:
+                    data.append(dict(np.load(fl, allow_pickle=True)))
+                except TypeError:
+                    # TODO: need to better handle this
+                    data.append(dict(np.load(fl, allow_pickle=True).item()))
```

Comment:
what's happening here?

Reply:
hmm... I think it was a numpy version thing that my forest data has to have .item() when load in

Reply:
are you sure it shouldn't be `.items()`? Is there a test of this?